blocks/init/src/Blocks/components/modal/assets/index.js

```diff
@@ -0,0 +1,24 @@
+import domReady from '@wordpress/dom-ready';
+import manifest from './../manifest.json';
+import MicroModal from 'micromodal';
+
+domReady(() => {
+	const selector = `.${manifest.componentJsClass}`;
+	const elements = document.querySelectorAll(selector);
+	const body = document.getElementsByTagName('body')[0];
+
+	// Append all modals at the bottom of body.
+	[...elements].forEach((element) => {
+		body.append(element);
+	});
+
+	// Initialize MicroModal.
+	MicroModal.init({
```

Comment:
please use js selectors to open and close modal just so If you want to change the library later on the selectors can stay the same.